### PR TITLE
[edit] API v1 use cases: basic content edits and added description

### PR DESCRIPTION
**Pull request by Catografix**
### Description

[edit] basic content edits and added description to API v1 use cases doc

### CODE REVIEW

**Comment by Catografix** [comment]
@khorne3 Hi Katie, would you like to review my changes?

File: articles/api/v1/use-cases.md [hunk 1]
```diff
@@ -11,28 +17,28 @@ Currently, there are certain features and functionality that are only available
...
+In API v1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
 
 `/api/connections/{AUTH0_CONNECTION}/socket`
 
 ## Client Users
 
-With APIv1, after authenticating with the `client_id` and the `client_secret` of an application, you may make a `GET` call to the [appropriate Users endpoint](https://auth0.com/docs/api/v1#!#get--api-clients--client-id--users) to return only those users that belong to a Connection enabled on that application.
+With API v1, after authenticating with the `client_id` and `client_secret` of an application, you can make a `GET` call to the [appropriate Users endpoint](/api/v1#!#get--api-clients--client-id--users) to return only those users that belong to any specified client connection that is enabled for that application.
 
 ## Email
```
Comment (Catografix) [inline articles/api/v1/use-cases.md]: can we link to this endpoint? I couldnt locate it.

File: articles/api/v1/use-cases.md [hunk 2]
```diff
@@ -11,28 +17,28 @@ Currently, there are certain features and functionality that are only available
...
 
-In APIv1, as part of your automation process, you may use the `PATCH` email endpoint to update email templates.
+With API v1, you can use the `PATCH` email endpoint to update email templates as part of your automation process.
 
 ## Enterprise Users/Directory Searching
 
-APIv1 allows you to search directly for users authenticating using Enterprise connections, such as Active Directory or Azure Active Directory.
+API v1 allows you to search directly for users authenticated using enterprise connections, such as Active Directory or Azure Active Directory.
```
Comment (Catografix) [inline articles/api/v1/use-cases.md:38]: Also this one...

File: articles/api/v1/use-cases.md [hunk 3]
```diff
@@ -11,28 +17,28 @@ Currently, there are certain features and functionality that are only available
...
 
 ## Client Users
 
-With APIv1, after authenticating with the `client_id` and the `client_secret` of an application, you may make a `GET` call to the [appropriate Users endpoint](https://auth0.com/docs/api/v1#!#get--api-clients--client-id--users) to return only those users that belong to a Connection enabled on that application.
+With API v1, after authenticating with the `client_id` and `client_secret` of an application, you can make a `GET` call to the [appropriate Users endpoint](/api/v1#!#get--api-clients--client-id--users) to return only those users that belong to any specified client connection that is enabled for that application.
 
 ## Email
 
-In APIv1, as part of your automation process, you may use the `PATCH` email endpoint to update email templates.
+With API v1, you can use the `PATCH` email endpoint to update email templates as part of your automation process.
```
Comment (Catografix) [inline articles/api/v1/use-cases.md:33]: Can we link to this endpoint? I couldnt locate it.

File: articles/api/v1/use-cases.md [hunk 4]
```diff
@@ -11,28 +17,28 @@ Currently, there are certain features and functionality that are only available
 
 ## Active Directory Connector Monitoring
 
-In APIv1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
+In API v1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
 
 `/api/connections/{AUTH0_CONNECTION}/socket`
```
Comment (Catografix) [inline articles/api/v1/use-cases.md:23]: Is there a url to link to for this endpoint?

**Comment by Catografix** [comment]
Looks good in the preview. Is there a way to indicate that Ive reviewed the preview for a PR or is that unnecessary?

File: articles/api/v1/use-cases.md (hunk 3 above)
Comment (khorne3) [inline articles/api/v1/use-cases.md:33]: Sorry, this is a PUT:

https://auth0.com/docs/api/v1#put--api-emails--email-template-name-

File: articles/api/v1/use-cases.md (hunk 4 above)
Comment (khorne3) [inline articles/api/v1/use-cases.md:23]: No, this is an undocumented endpoint (please see https://github.com/auth0/api2/issues/842 for more info).

File: articles/api/v1/use-cases.md (hunk 2 above)
Comment (khorne3) [inline articles/api/v1/use-cases.md:38]: All users from a specific directory: https://auth0.com/docs/api/v1#get--api-connections--connection--users
Specific users from a given directory: https://auth0.com/docs/api/v1#get--api-connections--connection--users-search--criteria-
All users from all directories: GET /api/enterpriseconnections/users?search={criteria}Search users from all enterprise directories

**Comment by Catografix** [comment]
Ready

**Pull request merged**